code/scripting/api/libs/ui.cpp

```diff
@@ -2,6 +2,8 @@
 //
 #include "ui.h"
 
+#include "freespace.h"
```

Comment:
This is a problem. We _really_ don't want to include freespace.h from _any_ file within the code folder. Do you really need this? I can't find the point in this file that is relying on some freespace.h declared value or function.

Reply:
That is for line 717, specifically the `flRealframetime` value

Reply:
I see. and other libs seem to do it as well like this. Meh.
Not a fan of how this is handled, but for this specific value it's ok I guess based on precendence